firestore.rules

```diff
@@ -1,8 +1,25 @@
 rules_version = '2';
 service cloud.firestore {
   match /databases/{database}/documents {
-    match /{document=**} {
-      allow read, write: if true;
-    }
+        match /items/{itemId} {
+          allow create: if request.auth != null;
+          allow update, delete: if request.auth.uid == request.resource.data.userId;
```

Comment:
We should also check that the userId on the existing item (`resource.data`) matches
```suggestion
          allow update, delete: if request.auth != null && request.auth.uid == request.resource.data.userId && request.auth.uid == resource.data.userId;
```